Runtime/Scripts/MeshSyncServer.Properties.cs

```diff
@@ -173,19 +166,27 @@ void SendUpdatedProperties()
 #endif
                 }
 
-                if (needsClientSync)
-                {
+                if (needsPythonCallback) {
+#if VERBOSE_LOGS
+                    Debug.Log("Sending changes, needed python callback.");
```

Comment:
same about updating the name and the log prefix

Reply:
Done